disass.ml

```diff
@@ -0,0 +1,38 @@
+open Instr
+
+let disassemble prog =
```

Comment:
My recommendation would be to print the string to a buffer ([Buffer](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Buffer.html) module documentation), which allows you to use an imperative "print this then that" style and is also more efficient than repeated string concatenations (not that it's really an issue here). You can use `Buffer.add_{char,string}` directly, but `let print = Printf.bprintf buf in .... print "%d" i .. print "%B" b` is probably more convenient (see [Printf](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Printf.html) documentation).